jujugui/static/gui/src/app/components/status/status.js

```diff
@@ -101,6 +112,18 @@ class Status extends React.Component {
   }
 
   /**
+    Filter a row by the status.
+    @param row {Object} The row values.
+    @returns {Boolean} Whether the row matches the status.
+  */
+  _filterByStatus(row) {
+    if (!this.state.filter) {
+      return true;
+    }
+    return row.filterValue === this.state.filter;
```

Comment:
if this.state.filter is a value, let's call it statusValue, or statusFilter, or statusFilterValue? ;-)